util/arbmath/bits.go

```diff
@@ -11,13 +11,13 @@ import (
 
 type bytes32 = common.Hash
 
-// flips the nth bit in an ethereum word, starting from the left
+// FlipBit flips the nth bit in an ethereum word, starting from the left
 func FlipBit(data bytes32, bit byte) bytes32 {
 	data[bit/8] ^= 1 << (7 - bit%8)
 	return data
 }
```

Comment:
Unrelated to your changes, but this method can panic if `bit/8` is greater than `len(data)-1`

Reply:
The length of the data is always 32 here but it'd probably be worth leaving a comment that bit must be less than 256